app/assets/javascripts/form_errors.coffee

```diff
@@ -0,0 +1,21 @@
+$(document).on "turbolinks:load", ->
+  new FormErrorHandler().reset()
+  new FormErrorHandler().execute()
+
+class @window.FormErrorHandler
```

Comment:
When using @window It will be placed on the `window` namespace, so It's available in view JS (create.js.erb) as well.